test/e2e/kappcontroller/sops_age_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
this is a thinly modified copy+paste of the existing sops test,  but it uses age.